python_modules/automation/automation/docker/image_defs.py

```diff
@@ -200,7 +200,7 @@ def dagster_celery_k8s_editable_cm(cwd: str) -> Iterator[None]:
 }
 
 
-def list_images(images_path: Optional[str] = None) -> List[DagsterDockerImage]:
+def list_images(images_path: str) -> List[DagsterDockerImage]:
```

Comment:
We talked offline but I think the bulk of this change could be represented by a change to the default on line 210.